### DISCUSSION TITLE
Adguard is asking for password on proxmox conso

### DISCUSSION THREAD

**Discussion by danielo515** [tteck/Proxmox#1892] [Help Needed] [answered]
It asks for default root password, but I didn't set any.
Which one it is?

**Comment by tteck** [comment, marked as answer]
If you didn't establish one using the Advanced settings, it suggests that the script didn't complete successfully. You must stay in the shell until you see "Completed Successfully!".

**Comment by danielo515** [comment]
Ok, that is probably the problem
